test/MemeTest.sol

```diff
@@ -37,6 +38,36 @@ contract MemeTest is TestBaseContract {
         proxy.buyMeme(MemeBuySizeDollars.Five);
     }
 
+    function testZkSyncMockPurchaseMeme() public {
```

Comment:
Change to `testPurchaseMemeWithMockTwap`. And change the following method to `testPurchaseMemeWithLiveTwap`.

`testBuyMemeWithInsufficientWMatic` should be replaced with one which uses the mock to set a price that isn't then met when calling buy.

Reply:
fixed, and meme tests are now dealing with coin instead of erc20 tokens

Reply:
Please rename the method.

Reply:
done